modules/oe_whitelabel_starter_event/oe_whitelabel_starter_event.module

```diff
@@ -171,3 +171,38 @@ function _oe_whitelabel_starter_event_preprocess_inpage_nav(array &$variables):
     ];
   }
 }
+
+/**
+ * Helper function to preprocess the date field.
+ *
+ * @param array $variables
+ *   Render array variables.
+ */
+function _oe_whitelabel_starter_event_preprocess_date(array &$variables): void {
```

Comment:
I didn't create a formatter (even though I wanted to) because it would use the date_block pattern  and that is not ready for date ranges.